packages/webapp/src/components/Badge/styles.module.scss

```diff
@@ -0,0 +1,73 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+@import '../../assets/mixin.scss';
+
+.tooltip {
+  color: #000;
+  border-radius: 4px;
+  font-size: 16px;
+  padding: 8px;
+  border: 1px solid var(--Colors-Neutral-Neutral-50);
+  box-shadow: 0px 4px 12px 0px #00000040;
+  z-index: 10;
+}
+
+.badge {
+  color: var(--Colors-Accent-Accent-yellow-600);
+  background: #fffbf2;
+  border-radius: 16px;
+  height: 24px;
+  font-size: 14px;
+  padding: 5px 4px 5px 6px;
+  > svg {
+    margin-left: 3px;
+  }
+  &:hover {
+    background: #ffeab2;
+    color: var(--Colors-Accent-Accent-yellow-700);
+  }
+  &:focus {
+    background: var(--Colors-Accent-Accent-yellow-500);
+    color: var(--Colors-Accent-Accent-yellow-50);
+    border-radius: 8px 8px 4px 4px !important;
+  }
+}
+
+:global(.hasBadge) .badge {
+  height: 12px;
+  font-size: 11px;
+  padding: 6px;
+  position: absolute;
+  left: 50px;
+  top: 8px;
+  border-radius: 16px;
+  &:focus,
+  &:hover {
+    color: var(--Colors-Accent-Accent-yellow-600);
+    background: #fffbf2;
+    border-radius: 16px !important;
+  }
+
+  @include xs-breakpoint {
+    height: 16px;
+    position: relative;
+    font-size: 14px;
+    top: initial;
+    left: initial;
+    padding: 6px;
+    margin-right: 6px;
+  }
+}
```

Comment:
Would it be a bad idea to pass a className with these styles to the Badge component instead?
I've never seen a raw CSS className used here before, so it would be nice if we could keep things consistent!

Reply:
It is one of the way by which we can access the parent class in the child component. But, i will see these styles can be attained by passing class to the component and also avoiding the inline css.